MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -137,6 +137,13 @@
 #include <setjmp.h>
 #include <unistd.h>
 
+jmp_buf jbAnagram;
+
+void *memset(void *s : byte_count(n), int c, size_t n) : bounds(s, (char *) s + n);
```

Comment:
Oh, it's to do with https://gcc.gnu.org/onlinedocs/gcc/Object-Size-Checking.html  

Basically, in some cases the compiler typedefs out the definition of memset as a declaration, replacing it with a call to a function that can do more static analysis on its arguments.

I think the fix is to add the `_Unchecked` keyword to all the prototypes in `_builtin_stdio_checked.h`, and any declarations protected by `#if !defined (__APPLE__) && _FORTIFY_SOURCE > 0`.